app/controllers/volunteers_controller.rb

```diff
@@ -72,6 +79,7 @@ def set_volunteer
 
   def volunteer_params
     params.require(:volunteer).permit(volunteer_attributes, :rejection_type, :rejection_text,
-      :trial_period, :intro_course, :doc_sent, :bank_account, :evaluation)
+      :trial_period, :intro_course, :doc_sent, :bank_account, :evaluation, :flexible,
```

Comment:
Why are the availability attributes here? They are already included in the volunteer_attributes through a concern.